src/app/public/theming/fixtures/theme-if-test.component.html

```diff
@@ -0,0 +1,2 @@
+<div *skyThemeIf="'modern'" class="sky-theme-if-test">modern theme</div>
+<div *skyThemeIf="'default'" class="sky-theme-if-test">default theme</div>
```

Comment:
Check your [code style formatting](https://host.nxt.blackbaud.com/skyux-team-docs/style#component-html-tags). Should be something like this...
```
<div *skyThemeIf="'default'"
  class="sky-theme-if-test"
>
  default theme
</div>
```